_includes/pbc/all/install-features/202407.0/install-the-cart-notes-feature.md

```diff
@@ -0,0 +1,146 @@
+
+
+{% info_block errorBox %}
+
+This feature integration guide expects the basic feature to be in place.
+The current feature integration guide adds the Cart Notes Backend API and Dynamic cart page update functionality.
```

Comment:
https://release.spryker.com/features/view/42
Cart notes feature was merged with the cart feature. So IGs also should be merged

Reply:
Also see some unclear things
- Name of IG:  `Cart Notes Backend API + Dynamic cart page update functionality.`
- You mention about widget with new controller in `CartNoteWidgetAsyncRouteProviderPlugin`.

How it could be written together?